src/main/java/org/jenkinsci/plugins/cloudstats/CloudStatistics.java

```diff
@@ -475,6 +524,17 @@ protected void doRun() throws Exception {
         return ((TrackedItem) plannedNode).getId();
     }
 
+    private static @CheckForNull ProvisioningActivity.Id getIdFor(Node node) {
+        if (node instanceof Jenkins) return null;
+
+        if (!(node instanceof TrackedItem)) {
+            LOGGER.info("No support for cloud-stats-plugin by " + node.getClass());
```

Comment:
Addressed in https://github.com/jenkinsci/cloud-stats-plugin/commit/ba2f509e8147fd6322ae4b4b45fc2fff4b0742a6 and https://github.com/jenkinsci/cloud-stats-plugin/commit/e40d20af6b7d3bc3bdd6c898b6f116d008c5ab06.